src/resources/js/docs.js

```diff
@@ -15,11 +17,17 @@ $(function() {
 		// as another special case, highlight the modules link anywhere within it
 		$currentPageLink = $('main nav a[href="/docs/modules/"]');
 	}
-	$currentPageLink.addClass('current');
+	$currentPageLink.addClass("current");
 
 	// add anchor links, inspired by https://github.com/bryanbraun/anchorjs
-	$('article > h1[id], article > h2[id], article > h3[id], article > h4[id], article > h5[id], article > h6[id]').each(function() {
-		var $anchor = $('<a href="#'+this.id+'" class="anchor-link" title="Direct link">🔗</a>');
+	$(
+		"article > h1[id], article > h2[id], article > h3[id], article > h4[id], article > h5[id], article > h6[id]"
+	).each(function () {
+		var $anchor = $(
+			'<a href="#' +
+				this.id +
+				'" class="anchor-link" title="Direct link">🔗</a>'
+		);
```

Comment:
We can probably use a template literal instead here. I don't think we care about IE at this point lol, we can use ES6 stuff.

Reply:
100% and for anyone still using IE, they should really re-evaluate their browser life choices.  Thanks for the suggestion, I've added the change

Reply:
Ha, yeah, at this point I'm only concerned about the major browser engines :100: